lib/praxis/types/multipart_array.rb

```diff
@@ -150,6 +148,67 @@ def self.example(context=Attributor::DEFAULT_ROOT_CONTEXT, **options)
         example
       end
 
+
+      def self.describe(shallow=true, example: nil)
+        type_name = Attributor.type_name(self)
```

Comment:
The problem is `super` is `Attributor::Collection.describe`. What we really need here is a "super -super", but have to settle for duplicating `Attributor::Type`'s here.